src/EnergyPlus/api/datatransfer.py

```diff
@@ -248,13 +268,25 @@ def get_actuator_handle(
         """
         if isinstance(component_type, str):
             component_type = component_type.encode('utf-8')
+        elif not isinstance(component_type, bytes):
+            raise EnergyPlusException(
+                "`get_actuator_handle` expects `component_type` as a `str` or UTF-8 encoded `bytes`, not "
+                "'{}'".format(component_type))
         if isinstance(control_type, str):
             control_type = control_type.encode('utf-8')
+        elif not isinstance(control_type, bytes):
+            raise EnergyPlusException(
+                "`get_actuator_handle` expects `component_type` as a `str` or UTF-8 encoded `bytes`, not "
+                "'{}'".format(control_type))
         if isinstance(actuator_key, str):
             actuator_key = actuator_key.encode('utf-8')
+        elif not isinstance(actuator_key, bytes):
+            raise EnergyPlusException(
+                "`get_actuator_handle` expects `component_type` as a `str` or UTF-8 encoded `bytes`, not "
+                "'{}'".format(actuator_key))
         return self.api.getActuatorHandle(component_type, control_type, actuator_key)
 
-    def get_variable_value(self, variable_handle: int) -> RealEP:
+    def get_variable_value(self, variable_handle: int) -> float:
```

Comment:
I changed the type hints for the entire file to use `float` and not `RealEP` as it should be.